src/libraries/Common/src/System/Security/Cryptography/SecKeyPair.cs

```diff
@@ -22,6 +24,9 @@ public void Dispose()
             PrivateKey = null;
             PublicKey?.Dispose();
             PublicKey = null!;
+            OwningCertificate?.Dispose();
```

Comment:
Doesn’t Dispose of SafeHandle have different characteristics from DangerousRelease? I find odd to pair it this way.

Reply:
Ah, yeah, you're right. I started off duplicating but changed to adding a reference.